### ISSUE TITLE
(FED) Update Twitter icon in footer

### ISSUE THREAD

**Issue by KirstyScott** [visitscotland/vs-component-library#81]
Twitter has rebranded and their logo is now different. This task asks to update the icon in the footer so that it uses the new twitter brand.

**Comment by duncansanderson** [comment]
Hey @kyle-fraser, can you create the new icon for X and upload it to the VS Font Awesome kit?

**Comment by duncansanderson** [comment]
I've updated the component library to include all Font Awesome icons, this means we can now use any Font Awesome icon as well as our custom icons.

The twitter icon the footer has been replaced by the X (x-twitter) icon. I've also replaced the twitter icon with the social share button component.

@CorinaMala these changes can be tested here:
- Footer: https://646df3398f07e38a95184dbb-gcqktyjagn.chromatic.com/?path=/story/footer--default
- Social share button: https://646df3398f07e38a95184dbb-gcqktyjagn.chromatic.com/?path=/story/social-share-button--open
